editquality/utilities/generate_make.py

```diff
@@ -4,20 +4,26 @@
 :Usage:
     generate_make -h | --help
     generate_make
-                 [--config-dir=<path>]
-                 [--diff]
-                 [--diff-only=<wiki_name>]
-                 [--output-path=<path>]
-                 [--template-dir=<path>]
+                 [--config=<path>]
+                 [--compare-to=<path>]
+                 [--compare-section=<name>]...
+                 [--output=<path>]
+                 [--templates=<path>]
+                 [--debug]
 
 :Options:
-    --config-dir=<path>     Directory to search for configuration files.
-    --diff                  Output diffs while generating.
-    --diff-only=<wiki_name> Only compare sections, don't generate.  Wiki name
-                            is given in English, for example "Spanish Wikibooks".
+    --config=<path>         Directory to search for configuration files
+                            [default: config/]
+    --compare-to=<path>     If specified, compare output to an old Makefile
```

Comment:
I like this change.  I found myself checking out old Makefiles for exactly the reason addressed here.